src/test-utils.ts

```diff
@@ -2,7 +2,8 @@ import CID from 'cids'
 import { CeramicService } from './services/ceramic-service'
 import { IpfsService } from './services/ipfs-service'
 import { StreamID, CommitID } from '@ceramicnetwork/streamid'
-import { MultiQuery, Stream } from '@ceramicnetwork/common'
+import { AnchorCommit, MultiQuery, Stream } from '@ceramicnetwork/common'
+import dagCBOR from 'ipld-dag-cbor'
```

Comment:
Ideally we should be using `@ipld/dag-cbor` but maybe we have to wait with that until we upgrade to the latest ipfs.